khelo/lib/ui/flow/stats/user_match/user_match_list_screen.dart

```diff
@@ -110,66 +75,97 @@ class _UserMatchListScreenState extends ConsumerState<UserMatchListScreen>
       },
       child: Container(
-        padding: const EdgeInsets.symmetric(horizontal: 16, vertical: 8),
-        margin: const EdgeInsets.symmetric(horizontal: 16),
+        padding: const EdgeInsets.all(16),
         decoration: BoxDecoration(
-          border: Border.all(color: context.colorScheme.outline),
-          borderRadius: BorderRadius.circular(20),
-          color: context.colorScheme.containerLow,
+          border: Border.all(color: context.colorScheme.secondary),
+          borderRadius: BorderRadius.circular(16),
         ),
         child: Column(
           crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            Row(
-              mainAxisAlignment: MainAxisAlignment.spaceAround,
-              children: [
-                Expanded(child: _teamNameView(context, match.teams.first.team)),
-                Text(
-                  context.l10n.add_match_versus_short_title,
-                  style: AppTextStyle.header4
-                      .copyWith(color: context.colorScheme.primary),
-                ),
-                Expanded(child: _teamNameView(context, match.teams.last.team)),
-              ],
+            _matchTimeAndGroundView(match),
+            const SizedBox(height: 16),
+            ...match.teams.map(
+              (e) => _teamView(context, e),
             ),
-            Divider(color: context.colorScheme.outline),
-            Text(
-              "${match.match_type.getString(context)} - ${context.l10n.match_list_overs_title(match.number_of_over)}",
-              style: AppTextStyle.subtitle2
-                  .copyWith(color: context.colorScheme.textPrimary),
+            Padding(
+              padding: const EdgeInsets.symmetric(vertical: 16.0),
+              child: Divider(
+                color: context.colorScheme.outline,
+                height: 1,
+              ),
             ),
-            Text(match.ground,
-                style: AppTextStyle.subtitle2
-                    .copyWith(color: context.colorScheme.textPrimary)),
-            Text(match.start_time.format(context, DateFormatType.dateAndTime),
-                style: AppTextStyle.subtitle2
-                    .copyWith(color: context.colorScheme.textPrimary)),
-            Divider(color: context.colorScheme.outline),
             _winnerMessageText(context, match)
           ],
         ),
       ),
     );
   }
 
-  Widget _teamNameView(BuildContext context, TeamModel team) {
-    return Column(
-      children: [
-        ImageAvatar(
-          initial: team.name[0].toUpperCase(),
-          imageUrl: team.profile_img_url,
-          size: 50,
-        ),
-        const SizedBox(
-          height: 4,
-        ),
-        Text(
-          team.name,
-          overflow: TextOverflow.ellipsis,
-          textAlign: TextAlign.center,
-          maxLines: 3,
+  Widget _matchTimeAndGroundView(MatchModel match) {
+    return Row(mainAxisAlignment: MainAxisAlignment.spaceBetween, children: [
```

Comment:
let's indent the code

Reply:
it's dart format. so i think no need to indent.